diff --git a/node_modules/react-native-spotify-remote/RNSpotifyRemote.podspec b/node_modules/react-native-spotify-remote/RNSpotifyRemote.podspec
index 4ac5f28..a88a93c 100644
--- a/node_modules/react-native-spotify-remote/RNSpotifyRemote.podspec
+++ b/node_modules/react-native-spotify-remote/RNSpotifyRemote.podspec
@@ -13,9 +13,9 @@ Pod::Spec.new do |s|
   s.platform     = :ios, "9.0"
 
   s.source       = { :git => package['repository']['url'], :tag => "v#{s.version}", :submodules => true }
-  s.source_files  = "ios/*.{h,m}","ios/external/SpotifySDK/SpotifyiOS.framework/**/Headers/*.{h,m}"
-  s.preserve_path = "ios/external/SpotifySDK/SpotifyiOS.framework"
-  s.vendored_frameworks = "ios/external/SpotifySDK/SpotifyiOS.framework"
+  s.source_files  = "ios/*.{h,m}"
+  s.preserve_path = "ios/external/SpotifySDK/SpotifyiOS.xcframework"
+  s.vendored_frameworks = "ios/external/SpotifySDK/SpotifyiOS.xcframework"
 
   s.dependency 'React-Core'
   
